src/main/resources/application.yml

```diff
@@ -33,6 +33,10 @@ micronaut:
   bi:
     api:
       version: v1
+  brapi:
+    no-version: brapi
+    v1: brapi/v1
+    v2: brapi/v2
```

Comment:
Why is this under `micronaut` and not at the root level?  This does apply to the `bi.api.version` property on lines 33-35, but that's outside of the scope of this PR.

Reply:
Ya, I was debating that. Just put it there because that's where the other version was but I can move it.

Reply:
Removed this from config as versions were moved to code. Pushed changes.